app/controllers/media_controller.rb

```diff
@@ -0,0 +1,59 @@
+##
+# API for delivering streaming media via stacks
+class MediaController < ApplicationController
+  before_action :load_media
+
+  rescue_from ActionController::MissingFile do
+    render text: 'File not found', status: :not_found
+  end
+
+  def download
+    return unless stale?(cache_headers)
```

Comment:
This seems to be copied over from file.  Does it make sense here?  Should we remove it until we have a reason to re-implement?